internal/pkg/cli/deploy/static_site.go

```diff
@@ -143,3 +148,20 @@ func (d *staticSiteDeployer) stackConfiguration(in *StackRuntimeConfiguration) (
 	}
 	return conf, nil
 }
+
+func (d *staticSiteDeployer) validateAlias() error {
+	if d.staticSiteMft.Alias == "" {
+		return nil
+	}
+
+	hasImportedCerts := len(d.envConfig.HTTPConfig.Public.Certificates) != 0
+	if hasImportedCerts {
```

Comment:
I wonder if people can still technically use an alias under `*.app.domain.com`, that has nothing to do with the env 🤔

Reply:
that's a great question... maybe for now i'll leave it as is (matching NLB logic) but if we have extra time we can see if it works with the cert requesting/validation.